code/modules/antagonists/fugitive/old_god.dm

```diff
@@ -186,10 +186,10 @@
 		to_chat(owner, "There are no followers left to heal!")
 		return
 	for(var/mob/living/A in faithful)
-		A.adjustBruteLoss(-20)
-		A.adjustFireLoss(-20)
-		A.adjustOxyLoss(-20)
-		A.adjustToxLoss(-20)
+		A.adjustBruteLoss(-20. TRUE, TRUE)
+		A.adjustFireLoss(-20. TRUE, TRUE)
+		A.adjustOxyLoss(-20. TRUE, TRUE)
+		A.adjustToxLoss(-20. TRUE, TRUE)
```

Comment:
```suggestion
		A.adjustBruteLoss(-20, TRUE, TRUE)
		A.adjustFireLoss(-20, TRUE, TRUE)
		A.adjustOxyLoss(-20, TRUE, TRUE)
		A.adjustToxLoss(-20, TRUE, TRUE)
```
Whoops used . instead of ,